apps/backend/src/routes/home/+page.svelte

```diff
@@ -0,0 +1,240 @@
+<!-- JavaScript code -->
+<script lang='ts'>
+  //Script to obtain current year
+  const currentYear = (new Date().getFullYear() as unknown) as string;
+
+  //Script to obtian Previous year
+  const previousYear = ((new Date().getFullYear() - 1) as unknown) as string;
```

Comment:
DO NOT DO THIS, MASSIVE ISSUE! `.getFullYear()` already returns a `number` type. All you need is to do `.toString()` to convert it to a `string`.

Reply:
Also, every time you do `new Date()`, it initialises a new `Date` object in memory, which will build up over time if you keep initialising new ones.

Save the `new Date()` object to a const variable and use it from there.

Reply:
Changed accordingly:
![Uploading image.png…]()

Reply:
Changed Accordingly:

```
// Script to create a Current Date object
const currentDate = new Date();

//Script to obtain current year
const currentYear = currentDate.getFullYear().toString();

//Script to obtian Previous year
const previousYear = (currentDate.getFullYear() - 1).toString();
```